services/leaf_space_integration/leaf_space_connections/main.go

```diff
@@ -0,0 +1,16 @@
+package leaf_space_connections
+
+import (
+	"fmt"
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var LeafSpaceIntegrationDataType = provider.DataSourceType[LeafSpaceConnection, *LeafSpaceConnection]{
+	ResourceIdentifier: "leanspace_leaf_space_integrations",
+	Path:               "integration-leafspace/connections",
+	Schema:             leafSpaceConnectionSchema,
+	FilterSchema:       nil,
```

Comment:
There's no find endpoint so I just tested and it doesn't work so either add the possibility to disable this (probably here: https://github.com/leanspace/terraform-provider-leanspace/blob/main/provider/generic_data_source.go#L21, this will also require not to generate the examples/data_source for this specific terraform resource) or override the schema which is probably as complex, maybe check with the team if they plan to keep it like that or not
```
data "leanspace_leaf_space_integrations" "all" {
}

output "all" {
  value = data.leanspace_leaf_space_integrations.all
}
```